codespell_lib/data/dictionary.txt

```diff
@@ -8536,30 +8650,52 @@ capabilitieis->capabilities
 capabilitiies->capabilities
 capabilitires->capabilities
 capabilitiy->capability
+capabillities->capabilities
 capabillity->capability
 capabilties->capabilities
+capabiltities->capabilities
 capabiltity->capability
 capabilty->capability
 capabitilies->capabilities
+capabitily->capability
 capablilities->capabilities
+capablility->capability
 capablities->capabilities
 capablity->capability
+capablle->capable
 capaciy->capacity
 capalize->capitalize
 capalized->capitalized
 capapbilities->capabilities
-capatibilities->capabilities
+capapbility->capability
+capapble->capable
+capapcity->capacity
+capatibilities->capabilities, compatibilities,
+capatibility->compatibility, capability,
+capatible->compatible, capable,
+capaticity->capacity
```

Comment:
Perhaps `captivity`, as **C** and `V` are neighbours in QWERTY keyboards.

Reply:
I couldn't find `capaticity->captivity`